### PR TITLE
feat(widget-chat): Add functional Activity List, messages and base styles

### PR DESCRIPTION
**Pull request by bzang**

### CODE REVIEW

File: packages/widget-chat/src/components/activity-item/index.js [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
   return (
-    <div>
-      <li>
-        {id} - {content}
-      </li>
+    <div className={classNames(`activity-item`, styles.activityItem)}>
+      <div className={classNames(`avatar-wrapper`, styles.avatarWrapper)}>
+        <Avatar name={name} />
+      </div>
+      <div className={classNames(styles.content)}>
```
Comment (adamweeks) [inline packages/widget-chat/src/components/activity-item/index.js]: At what point do we stop giving overridable class names?

File: packages/widget-chat/src/components/avatar/styles.css [hunk 2]
```diff
@@ -7,7 +7,7 @@
   background-color: darkgray;
   width: 40px;
   height: 40px;
-  border-radius: 20px;
+  border-radius: 50%;
```
Comment (adamweeks) [inline packages/widget-chat/src/components/avatar/styles.css]: Nice!

File: packages/widget-chat/src/components/activity-item/index.js (hunk 1 above)
Comment (bzang) [inline packages/widget-chat/src/components/activity-item/index.js]: Good question. This is more of an oversight on my part than anything else. But the larger discussion point is, do we want to make everything overridable? Or just some things. At this point I think it's fine to make all of the things overridable.

Comment (ianwremmel) [inline packages/widget-chat/src/components/activity-item/index.js]: To that end, can we make our own wrapper around `classNames` that takes the overridable name in the first position?

Comment (bzang) [inline packages/widget-chat/src/components/activity-item/index.js]: The overridable name is already in the first position, right? The additional params are any styles we want to attach internally.

Comment (ianwremmel) [inline packages/widget-chat/src/components/activity-item/index.js]: i was thinking something that would warn us if we forgot the overridable part

Comment (bzang) [inline packages/widget-chat/src/components/activity-item/index.js]: ahhh. is there a way for us to distinguish between a override class string and the string that's in styles.whatever?

**Pull request merged**